src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -4,7 +4,7 @@
 	overflow: hidden;
 	top: 0;
 	left: 0;
-
+	min-height: 250px;
```

Comment:
As I see in this file all values base on the @unit_base
e.g.
width: 360 * @unit_base;
height: 108 * @unit_base;

I think we should use it here also

Reply:
Ok, I will change it. Do you know if `@unit_base` is the similar to pixel?

Reply:
Yes, I think this was intended as pixel independent

Reply:
Done!